commit 68e95495e6be730daabe244f165a5d92b8529bba (refs/changes/14/5514/1)
Author: Tim Foster <tim.foster@joyent.com>
Date:   2019-02-08T09:40:17+00:00 (8 months ago)
    
    TRITON-1177 hermes build looking for node in all the wrong places

diff --git a/deps/hermes b/deps/hermes
index d3c3ea8..5760e25 160000
--- a/deps/hermes
+++ b/deps/hermes
@@ -1 +1 @@
-Subproject commit d3c3ea81fffb0df23d22ede731d2e6bdd7384073
+Subproject commit 5760e25e2386b9ab4e7f61ee2bf5aed4eca1512b
